package-system/expat/test/example_file.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<rootelement>
+    <!-- a comment in here and a utf-8 element. This is not an 
+    exhaustive self test of expat though, it has those in its own code. -->
+    <special-element>✔️</special-element>
```

Comment:
this may break if we ever introduce the validation scripts into these jobs, I suggest using the escape sequence if possible

Reply:
Escape sequence?   This is a utf-8 encoded file with utf text inside it.   Woudln't escaping it reduce the value of making sure expat can read actual utf-8 encoded files?

Reply:
I see... I was thinking that is not a great test for our case because we dont allow these characters in xml files (we have a validation check). 
I am ok with leaving it, just something to be aware if we want to run the validation scripts in this repo (those validations may be useful since we have the legal header validations there)